.gitignore

```diff
@@ -1,4 +1,5 @@
-/attachments/
+/attachments/*
+!/attachments/.gitkeep
```

Comment:
I disagree with that answer, based on this answer: http://stackoverflow.com/questions/7229885/what-are-the-differences-between-gitignore-and-gitkeep
Basically, .gitkeep is a convention thing, but doesn't really mean anything, whereas .gitignore has significance.  I think the .gitignore might be cleaner, but it's not too important.